eng/build.sh

```diff
@@ -542,6 +542,9 @@ fi
 if [[ "$os" == "browser" ]]; then
     # override default arch for Browser, we only support wasm
     arch=wasm
+    # because on docker instance without swap file, MSBuild nodes need to make some room for LLVM
```

Comment:
Can you add a slight explanation of how these numbers were chosen in case they turn out to be insufficient later or someone wants to reduce them? i.e. how did you test to see whether these were the right numbers, or what data did you use to choose them

Reply:
And it doesn't even always help. On this PR we have `##[warning]Free memory is lower than 5%; Currently used: 95.82%` and the agent would be killed bit later.

Reply:
I think we should at least file a tracking issue in dotnet/runtime and link it here so that we reevaluate later

Reply:
https://github.com/dotnet/runtime/issues/113724

Reply:
I reverted that and now I only allow msbuild to run single node